src/pages/bridge/components/TxTable/index.tsx

```diff
@@ -192,8 +226,88 @@ const TxRow = props => {
     )
   }
 
+  const handleSwitchNetwork = async (chainId: number) => {
+    try {
+      // cancel switch network in MetaMask would not throw error and the result is null just like successfully switched
+      await switchNetwork(chainId)
+    } catch (error) {
+      // when there is a switch-network popover in MetaMask and refreshing page would throw an error
+    }
+  }
+
+  const renderClaimButton = tx => {
+    if (tx.isL1 || fromStatus === TX_STATUS.failed) return null
+    if (tx.isClaimed) {
+      return <Chip label="Claimed" className={cx(classes.chip, classes.claimedChip)} />
+    }
+
+    const isOnScrollLayer1 = chainId === CHAIN_ID.L1
+    if (tx.isFinalized) {
+      if (isOnScrollLayer1) {
+        return (
+          <Button className={classes.claimButton} onClick={() => handleClaim(tx.claimInfo)} disabled={loading} color="primary" variant="contained">
+            Claim {loading ? <CircularProgress size={16} sx={{ position: "absolute", right: "0.8rem" }} color="inherit" /> : null}
+          </Button>
+        )
+      } else {
+        return (
+          <Tooltip placement="top" title="Please connect to the L1 network to claim your withdrawal.">
+            <Box>
+              <Button
+                className={classes.claimButton}
+                onMouseEnter={() => setClaimButtonLabel("Switch")}
+                onMouseLeave={() => setClaimButtonLabel("Claim")}
+                onClick={() => handleSwitchNetwork(CHAIN_ID.L1)}
+              >
+                {claimButtonLabel}
+              </Button>
+            </Box>
+          </Tooltip>
+        )
+      }
+    } else {
+      return (
+        <Tooltip
+          placement="top"
+          title="Our provers are still finalizing your transaction on Scroll ALPHA Testnet, when it has finalized you can come back and claim your transaction. This usually takes 1-4 hours after your transaction has been submitted on the bridge."
```

Comment:
```suggestion
          title="Scroll provers are still finalizing your transaction, this can take up to 4 hours. Once done, you'll be able to claim it here for use on the target network."
```